### DISCUSSION TITLE
Daily 09/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#251] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy
- Dificultades o Impedimentos

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve creando las tareas en el repositorio que habíamos acordado y algunas más.
Hoy voy a estar leyendo la parte teórica, de la plataforma, de los temas para este sprint.

**Comment by rpgrca** [comment]
Ayer fui al médico y estuve con la cabeza a mitad de revoluciones todo el día, sin embargo pude ir a la clase de la tarde y después en la llamada de la noche expliqué de manera simple lo de los servicios. También terminé con las tareas de servicios de productos como ejemplo.

**Comment by BraianTroncoso** [comment]
Ayer realicé unas clases de Python y Js sobre OOP. También tuve ingles y asistí al meet para organizarnos sobre este sprint y vimos sobre servicios.
Estuve con poco tiempo entre el trabajo y sumado que cumplí años anduve a mil pero llegué con todo.
